tests/cypress.config.js

```diff
@@ -42,6 +42,7 @@ module.exports = defineConfig({
       'tests/namespace/test-oauth2.spec.js',
       'tests/namespace/test-dns-entries.spec.js',
       'tests/namespace/test-services.spec.js',
+      'tests/namespace/test-service-bindings.spec.js',
```

Comment:
I couldn't find this tests.
Should it be removed?

Reply:
Yes, it should, it's now removed